community/CONTRIBUTING.md

```diff
@@ -65,45 +76,69 @@ Thanks for considering contributing to OPEA project. The contribution process is
     GenAIComps
     ├── comps
     │   └── embeddings
-    │       ├── __init__.py
-    │       └── tei     #vendor name or serving framework name
-    │           ├── langchain
-    │           │   ├── Dockerfile
-    │           │   ├── Dockerfile.amd_gpu
-    │           │   ├── Dockerfile.nvidia_gpu
-    │           │   ├── embedding_tei.py    # definition and registration of microservice
-    │           │   ├── README.md
-    │           │   └── requirements.txt
-    │           └── llama_index
-    │               └── . . .
+    |   |   ├── deployment
+    |   |   │   ├── docker_compose
+    |   |   │   │   └── compose.yaml
+    |   |   │   └── kubernetes
+    |   |   │       ├── README.md
+    |   |   │       └── cpu-values.yaml
+    |   |   └── src
+    |   |       ├── Dockerfile
+    |   |       ├── README.md    # could be multiple files for different integrations
+    |   |       ├── __init__.py
+    |   |       ├── integrations
+    |   |       │   ├── __init__.py
+    |   |       │   ├── multimodal_bridgetower.py
+    |   |       │   ├── predictionguard.py
+    |   |       │   └── tei.py
+    |   |       ├── opea_embedding_microservice.py
+    |   |       ├── opea_multimodal_embedding_microservice.py
+    |   |       └── requirements.txt
+    |   ├── third_parties
+    |   │   ├── bridgetower
+    |   │   ├── clip
+    |   │   ├── elasticsearch
+    |   │   ├── gpt-sovits
+    |   │   ├── milvus
+    |   │   ├── mongodb
+    |   │   ├── nginx
+    |   │   ├── opensearch
+    |   │   ├── pathway
+    |   │   ├── pgvector
+    |   │   ├── redis
+    |   │   ├── speecht5
+    |   │   ├── tei
+    |   │   ├── teirerank
+    |   │   ├── tgi
+    |   │   ├── vdms
+    |   │   ├── vllm
+    |   │   ├── wav2lip
+    |   │   └── whisper
     ├── tests
     │   └── embeddings
     │       ├── test_embeddings_tei_langchain.sh
```

Comment:
Those test need to update, the test name should be 'test_[microservice type]_[integration type].sh' or 'test_[microservice type]_[integration type]_on_[hardware].sh'.